### DISCUSSION TITLE
Remove xx days

### DISCUSSION THREAD

**Discussion by MikeAnita** [totaldebug/atomic-calendar-revive#1355] [Support]
Hello,
Can someone please tell me how to remove this entry?
I don't want an ad in xx days.
I think I'm too stupid for this, I can't find it.

Thank you so much
![calendar](https://github.com/totaldebug/atomic-calendar-revive/assets/103495472/948461e9-efc4-487d-8477-931ac68a76cc)

**Comment by marksie1988** [comment]
That is the relative time setting, set it to false 🙂

**Reply by MikeAnita** [reply]
Thank you Soo much 🥇
